Taskfile.yml

```diff
@@ -21,6 +21,13 @@ tasks:
     cmds:
       - find ./* -type d -mindepth 0 -maxdepth 0 -print0 | xargs -0 helm lint --strict
 
+  install-chart-deps:
```

Comment:
Please pull the changes that needs approval from compliance team into a separate repo so I can approve just that small change rather than all 2000+ lines

Reply:
derp, not repo, I meant pull request.

Reply:
Moved this change to #4